components/display/Inkplate.rst

```diff
@@ -0,0 +1,242 @@
+Inkplate 6
+==========
+
+.. seo::
+    :description: Instructions for setting up Inkplate E-Paper displays in ESPHome.
+    :image: Inkplate.jpg
+
+All-in-one e-paper display  ``Inkplate 6``
+Inkplate 6 is a powerful, Wi-Fi enabled ESP32 based six-inch e-paper display – recycled from a Kindle e-reader. Its main feature is simplicity.
+Learn more at `our website <https://inkplate.io/>`__
+
+.. figure:: images/Inkplate.jpg
+    :align: center
+    :width: 75.0%
+
+    Inkplate 6
+
+
+.. code-block:: yaml
+
+    # Example minimal configuration entry
+
+    mcp23017:
+      - id: mcp23017_hub
+        address: 0x20
+
+    display:
+    - platform: inkplate
+      id: inkplate_display
+      greyscale: false
+      partial_updating: false
+      update_interval: 60s
+
+      ckv_pin: 32
+      sph_pin: 33
+      gmod_pin:
+        mcp23017: mcp23017_hub
+        number: 1
+      gpio0_enable_pin:
+        mcp23017: mcp23017_hub
+        number: 8
+      oe_pin:
+        mcp23017: mcp23017_hub
+        number: 0
+      spv_pin:
+        mcp23017: mcp23017_hub
+        number: 2
+      powerup_pin:
+        mcp23017: mcp23017_hub
+        number: 4
+      wakeup_pin:
+        mcp23017: mcp23017_hub
+        number: 3
+      vcom_pin:
+        mcp23017: mcp23017_hub
+        number: 5
+
+.. warning::
+
+    When using the Inkplate epaper module, the GPIO pin numbers above *cannot be changed* as they are
+    hardwired within the module/PCB.
+
+
+Configuration variables
+***********************
+
+- **name** (**Required**, string): The name of the switch.
+- **greyscale**: Makes the screen display 3 bit colors.
+- **partial_updating**: Makes the screen update partialy, which is faster, but leaves burnin. 
+- **lambda** (*Optional*, :ref:`lambda <config-lambda>`): The lambda to use for rendering the content on the display.
+  See :ref:`display-engine` for more information.
+- **update_interval** (*Optional*, :ref:`config-time`): The interval to re-draw the screen. Defaults to ``5s``.
+- **pages** (*Optional*, list): Show pages instead of a single lambda. See :ref:`display-pages`.
+- **id** (*Optional*, :ref:`config-id`): Manually specify the ID used for code generation.
+
```

Comment:
```suggestion

- **id** (*Optional*, :ref:`config-id`): Manually specify the ID used for code generation.
- **greyscale** (*Optional*, boolean): Makes the screen display 3 bit colors. Defaults to ``False``
- **partial_updating** (*Optional*, boolean): Makes the screen update partially, which is faster, but leaves burnin. Defaults to ``False``
- **full_update_every** (*Optional*, int): When partial updating is enabled, forces a full screen update after chosen number of updates. Defaults to ``10``
- **lambda** (*Optional*, :ref:`lambda <config-lambda>`): The lambda to use for rendering the content on the display.
  See :ref:`display-engine` for more information.
- **update_interval** (*Optional*, :ref:`config-time`): The interval to re-draw the screen. Defaults to ``5s``.
- **pages** (*Optional*, list): Show pages instead of a single lambda. See :ref:`display-pages`.

```